style/style.css

```diff
@@ -475,3 +463,271 @@ textarea {
+    height: 72px;
+    align-items: center;
+    background-color: #fff;
+    margin: 20px 155px;
+  }
+
+  .content {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    background-image: url(../assets/Headerbg.png);
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-color: rgb(255, 255, 255);
+    background-position: center;
+    height: 916px;
+    margin: 0 auto;
+  }
+
+  .content h1 {
+    font-size: 48px;
+    line-height: 52px;
+    max-height: 122px;
+    width: fit-content;
+    margin: 287px 18% 12px 18%;
+    justify-content: left;
+  }
+
+  .content div {
+    margin: 6px 18%;
+    justify-content: left;
+    width: 922px;
+  }
+
+  .content p {
+    margin: 0;
+    font-size: 20px;
+    line-height: 28px;
+  }
+
+  #connect {
+    margin: 0;
+  }
+
+  .content .social {
+    margin: 12px 18% 255px 18%;
+  }
+
+  .works {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    flex-basis: 0;
+    background-color: rgb(255, 255, 255);
+    min-height: 496px;
+    border-radius: 16px;
+    margin: 88px 20%;
+    padding: 1px 1px 1px 1px;
+    border: 1px solid #dfe1e6;
+    box-sizing: border-box;
+    width: fit-content;
+  }
+
+  .works:hover {
+    border: 2px solid #a7aeff;
+    border-radius: 24px;
+  }
+
+  .desktop-img {
+    width: 544px;
+    height: 448px;
+  }
+
+  .works img {
+    margin: 24px;
+  }
+
+  .big {
+    flex-direction: row-reverse;
+  }
+
+  .works div {
+    max-height: 492px;
+  }
+
+  .works h1 {
+    margin: 48px 46px 0 46px;
+    height: 48px;
+  }
+
+  .works ul {
+    margin: 20px 46px 22px 46px;
+  }
+
+  .works p {
+    font-size: 16px;
+    line-height: 24px;
+    margin: 10px 46px;
+  }
+
+  .works .languages {
+    margin: 18px 46px;
+  }
+
+  .works h2 {
+    margin: 45px 58px 0 40px;
+  }
+
+  .contact {
+    display: flex;
+    flex-direction: row;
+    height: 668px;
+  }
+
+  .about-myself {
+    margin: 142px 0 0 11%;
+    width: 31%;
+  }
+
+  .contact h1 {
+    font-size: 40px;
+    line-height: 52px;
+    margin: 46px 0 12px 24px;
+  }
+
+  .contact p {
+    font-size: 16px;
+    line-height: 24px;
+    height: 100px;
+    width: 68%;
+  }
+
+  #about-lenguages-container {
+    width: 47%;
+    margin: 142px 0 45px 0;
+  }
+
+  .lenguages-html {
+    width: 122px;
+    height: 120px;
+    display: flex;
+    flex-direction: column;
+    gap: 0;
+    align-items: flex-start;
+    margin: 0 12px 0 0;
+  }
+
+  .lenguages-html img {
+    margin: 0;
+    padding: 10px 0 16px 10px;
+  }
+
+  .lenguages-html h4 {
+    margin: 0;
+    padding: 0 0 0 10px;
+  }
+
+  #section-class {
+    display: flex;
+    flex-direction: row;
+    margin: 36px 37px;
+  }
+
+  #contact-me {
+    background-image: url(../assets/contact.background.desktop.png);
+    background-position: center;
+    min-height: 846px;
+    width: 100%;
+  }
+
+  #contact-me p {
+    width: 569px;
+    height: 102px;
+  }
```

Comment:
[OPTIONAL]

- Using ID much is not a good idea. Instead, you can use a class. Let's take a scenario:

suppose if you want to give some style to other places in the HTML file. then the benefit of using class is you can use that class in another place but not ID's. So, be careful while styling with ID's.